modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -56,8 +56,14 @@ Runtime Fabric enables you to specify the severity level of messages that are wr
 
 Log levels are specified per Mule application or API proxy during deployment.
 
-[WARNING]
-Because log levels are specified when deploying a Mule application or an API proxy, you cannot change the log levels after deployment.
+[IMPORTANT]
+
+* You can use Anypoint Runtime Manager to change the Runtime Fabric log level by configuring following property:
```

Comment:
_the_ following property?

Reply:
also, its more accurate to say "application log level" or "Mule log level" rather than "Runtime Fabric log level". You are changing it for a specific application, not globally for all applications